components/adblock_rust_ffi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Brian R. Bondy <netzen@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-adblock = { version = "0.6.2", default-features = false, features = ["full-regex-handling", "object-pooling", "unsync-regex-caching"] }
+adblock = { git = "https://github.com/brave/adblock-rust", rev = "5959733c38ad35f2639e435f5fc2d05695fdee67", default-features = false, features = ["full-regex-handling"] }
```

Comment:
This is https://github.com/brave/adblock-rust/tree/5959733c38ad35f2639e435f5fc2d05695fdee67, which is just a direct fork from `0.6.2` with a backported commit.

I initially created a patch version release (`0.6.4`), but that's actually not going to work because `0.6.3` required some other `brave-core` changes to work correctly.